### ISSUE TITLE
Unable to enter credentials: `fe_sendauth: no password supplied`

### ISSUE THREAD

**Issue by defel** [TablePlus/TablePlus-Linux#61]
1. Database version (Ex: PostgreSQL 10.0): irrelevant for this issue

2. TablePlus version (the number on the welcome or about screen, Ex: build 81):  alpha build 54

3. Linux distro (Ex: Ubuntu 18.04): arch linux

4. The steps to reproduce this issue:

see recording: 
![ezgif-3-7b59cb64e556](https://user-images.githubusercontent.com/131087/82160645-18190d00-9897-11ea-9e5e-319df3ee3dd9.gif)

:warning: **on the shell I get the following error displayed**:

> Error: Cannot create an item in a locked collectionlibsecret-Message: 23:12:16.610: Remote error from secret service: org.freedesktop.Secret.Error.IsLocked: Cannot create an item in a locked collection

Maybe this is related this this: https://github.com/r-lib/keyring/issues/47 

Note: I have a little bit special setup with GPG_AGENT and SSH_AGENT on my machine, maybe this breaks something on libsecret, which break the password saving, which breaks the profile and results in `fe_sendauth: no password supplied` :/ 

But what I would expect is, to use the created profile in-place. It would be ok when password is not saved as a work-around.

**Comment by huyphams** [comment]
Hi @defel please install this package and see it goes: https://www.archlinux.org/packages/extra/x86_64/gnome-keyring/

**Comment by defel** [comment]
Hi @huyphams 

thanks - I have gnome-keyring installed and it is started with `dbus.service` - unfortunately I'm unable to unlock the keyring with a gui (i.e. seahorse) - so I suspect the reason for this may be the same problem as here. 

I doubt that I am not able to solve the problem with gnome-key now - and I think this is a little out of scope of this issue.

**Comment by defel** [comment]
Ok, I have now dealt with this topic a little more. 

Short-Story: It was necessary to start the keyring-daemon correctly on login.

Long-Story: 
It looks like that this is more or less arch linux specific (or everyone else who logs in over terminal). I tried to use `seahorse` to unlock the password-store and notices that is was not possible to interact with the keyring. It failed without an error message, as before with tableplus or other programs.

Then I found this thread: https://unix.stackexchange.com/questions/265503/how-do-i-fix-no-such-secret-collection-at-path-for-gnome-keyring-and-arch-l

Starting the keyring-daemon as follows fixed the issue: 

```
systemctl --user import-environment DISPLAY XAUTHORITY
dbus-update-activation-environment --systemd DISPLAY XAUTHORITY
eval $(/usr/bin/gnome-keyring-daemon --start --components=pkcs11,secrets,ssh)
export SSH_AUTH_SOCK
```

ps.: For the search-engine: `"no such secret collection at path" /` (error in seahorse when trying to create a new password-store, when keyring-daemon is not running under the users-session correctly) 

pps.: My personal issue is fixed, tableplus works fine now - but decide you if you want to handle the situation when keyring-daemon is not running correctly and therefor tableplus is unable interact with libsecret. For me it would be ok to close this issue now.
